test/integration/apis/btc-core.test.ts

```diff
@@ -105,26 +105,53 @@ describe("BTCCore regtest", function () {
 
     let api: ApiPromise;
     let btcCore: BTCCoreAPI;
+    let bitcoinCoreClient: BitcoinCoreClient;
 
     beforeEach(async () => {
         api = await createPolkadotAPI(defaultEndpoint);
         btcCore = new DefaultBTCCoreAPI("http://0.0.0.0:3002");
+        bitcoinCoreClient = new BitcoinCoreClient(
```

Comment:
I would include this initialization in a  `before` block rather than the `beforeEach`, since `bitcoinCoreClient` holds no state besides the connection